src/form/DateTimeInput/DateTimeInput.tsx

```diff
@@ -179,7 +179,7 @@ export default function DateTimeInput(props: Props) {
           {label}{' '}
           <span
             className={`date-time-input-format ${
-              value && hasFormatError ? 'text-danger' : 'text-muted'
+              value && hasFormatError ? 'text-danger' : ''
             }`}
```

Comment:
It is better readable if we use `classNames` here.

```suggestion
            className={classNames('date-time-input-format', {
              'text-danger': value && hasFormatError
            })}
```